### PR TITLE
4 new table added

### PR DESCRIPTION
**Pull request by ekesgin**

### CODE REVIEW

File: datasets/huishoudelijkafval/rolcontainer/v1.0.0.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        "title": "Wijzigingsdatum"
+      },
+      "soortContainer": {
+        "type": "string",
+        "description": "Dit attribuut geeft een aantal kenmerken (materieel, inhoud in liter en fractie) van een rolcontainer. Bijvoorbeeld: GFE/T 140 Kunststof.",
+        "title": "Soort container"
+      },
+      "straatnaam": {
+        "type": "string",
+        "description": "Offici\u00c3\u00able naam openbare ruimte (80 tekens)",
```
Comment (RobbieLuchs) [inline datasets/huishoudelijkafval/rolcontainer/v1.0.0.json:89]: Please remove unicodes

File: datasets/huishoudelijkafval/rolcontainer/v1.0.0.json [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        "title": "Postcode"
+      },
+      "woonplaatsNaam": {
+        "type": "string",
+        "description": "Een Woonplaats is een door het bevoegde gemeentelijke orgaan als zodanig aangewezen en van een naam voorzien gedeelte van het grondgebied van de gemeente.",
+        "title": "BAG Woonplaats naam"
+      },
+      "geometrie": {
+        "$ref": "https://geojson.org/schema/Point.json",
+        "description": "Geometrie van het type POINT van de rolcontainer RD (epsg:28992). De geometrie\u00c3\u00abn zijn afkomstig uit BAG. ",
```
Comment (RobbieLuchs) [inline datasets/huishoudelijkafval/rolcontainer/v1.0.0.json:119]: Remove unicodes

**Review by RobbieLuchs — CHANGES_REQUESTED** [review]
Lots of unicodes cluttering the descriptions, please take a thorough look.

File: datasets/huishoudelijkafval/rolcontainer/v1.0.0.json (hunk 1 above)
Comment (ekesgin) [reply datasets/huishoudelijkafval/rolcontainer/v1.0.0.json:89]: aangepast

File: datasets/huishoudelijkafval/rolcontainer/v1.0.0.json (hunk 2 above)
Comment (ekesgin) [reply datasets/huishoudelijkafval/rolcontainer/v1.0.0.json:119]: aangepast

**Pull request closed without merging**